jobconfig.toml

```diff
@@ -235,20 +237,20 @@
 [ROBOTICIST]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 2
+"Spawn Positions" = 3
+"Total Positions" = 3
```

Comment:
Keep this at two, I feel like that's already a bit much.

Reply:
Understood; Reduced to 2.